docs/start/getting-started/fragments/flutter/integrate.md

```diff
@@ -115,3 +115,25 @@ void _recordEvent() async {
 ```
 
 At this point you are almost ready to run your app.  In the next section, we will use Amplify CLI to configure your backend AWS resources.
+
+## Using Flutter Hot Reload and Hot Restart
+
+Flutter offers [hot reload and hot restart](https://flutter.dev/docs/development/tools/hot-reload) functionality in order to aid the development process.
+
+A key difference between hot reload and restart is that hot restart destroys the state in dart, while hot reload does not.  Amplify-flutter should automatically re-configure your application and re-wire it's plugins upon a hot restart, so there is no additional effort needed on your part.
+
+Depending on how and when you are calling `Amplify.configure`, you may wish to check whether or not Amplify has already been configured because it should not be configured more than once.  
+
+To do this, you can use Amplify-flutter's `isConfigured` getter:
+
+```dart
+  void _isConfigured() {
```

Comment:
Even though (especially since?) this is example code, this function should probably either return a `bool` or be named `setConfigurationStatus`

Reply:
Done.